plugins/modules/lvm_facts.py

```diff
@@ -267,14 +267,14 @@ def load_pvs(module, name, LVM):
         name     (str): physical volume name.
         LVM     (dict): LVM facts.
     return:
-        msg  (str): message
-        LVM (dict): LVM facts
+        warnings (list): List of warning messages
+        LVM      (dict): LVM facts
     """
-    msg = ""
+    warnings = list()
     cmd = "lspv"
     rc, stdout, stderr = module.run_command(cmd)
     if rc != 0:
-        msg += "Command '%s' failed." % cmd
+        warnings.append(f"Command failed. {cmd=} {rc=} {stdout=} {stderr=}")
```

Comment:
Module doc lists requirement for `Python >= 2.7`. My patch requires >=3.6 for f-strings. If I took f-strings out there's also a dictionary unpack operator that requires >=3.5.

Is Python2 compatibility still being maintained? It has been deprecated for over 3 years now!

Reply:
f-strings are still used, but I removed all uses of var expansion (e.g. `f"{cmd=}"`) since that requires >= Python 3.8